README.md

```diff
@@ -338,25 +400,48 @@ v3c.write(backend="kv", table="mytable", dfs=df, expression="city='NY'", conditi
 - <a id="method-write-kv-param-condition"></a>**condition** (Optional) (default: `None`) &mdash; A platform condition expression that defines conditions for performing the write operation.
   For detailed information about platform condition expressions, see the [platform documentation](https://www.iguazio.com/docs/reference/latest-release/expressions/condition-expression/).
 
-Example:
+<a id="method-write-params-tsdb"></a>
+#### `tsdb` Backend `write` Parameters
```

Comment:
Might be worth specifying the "format" of the dataframe you need to save for tsdb, meaning:
1. There should be one index column of type time that will function as the timstamp for every sample
2. Any other index columns (in case of a multi index) will be used as the labels for that specific row.
3. Any "regular" column would be ingested as a separate metric

Reply:
ACCEPT — I edited the description of the `dfs` and `labels` parameters of the `write` method in the README file and in the help text. See commit 263852e.

(BTW, the `write` method wasn't part of this PR, other than the cosmetic changes done for all methods for consistency. There are many other changes to be done for the `write`, `delete`, and `execute` methods.)